src/main/java/com/thebois/ColonyManagement.java

```diff
@@ -13,15 +15,36 @@
 
     private ShapeRenderer batch;
     private BitmapFont font;
+    private final int worldSize = 50;
+
+    // Models
+    private World world;
+
+    // Views
+    private WorldView worldView;
+
+    // Screens
     private GameScreen gameScreen;
 
+    // Controllers
+
     @Override
     public void create() {
         batch = new ShapeRenderer();
 
         // use libGDX's default Arial font
         font = new BitmapFont();
-        gameScreen = new GameScreen(batch);
+
+        // Models
+        world = new World(worldSize);
+
+        // Views
+        worldView = new WorldView(world.getWorld());
+
+        // Screens
+        gameScreen = new GameScreen(batch, worldView, worldSize);
```

Comment:
We probably should change how gameScreen gets all the view in order to not have a extremly large constructor with all the views. 

A sulution would be to have a view interface that alls views implements with a draw method taking batch as a parameter. And then gameScreen would only take a list of the view interface. In the gameScreen inside the render method you would loop through all the views and call the draw method.